src/LoanActionsHistory.js

```diff
@@ -432,11 +518,11 @@ class LoanActionsHistory extends React.Component {
               formatter={loanActionsFormatter}
               visibleColumns={['actionDate', 'action', 'dueDate', 'itemStatus', 'Source']}
               columnMapping={{
```

Comment:
I think `columnMapping` right now has to receive strings - see https://github.com/folio-org/ui-data-import/pull/20 and https://issues.folio.org/browse/STCOM-399

Reply:
Oh.. I see to calculate the width;